docs/_docs/themes.md

```diff
@@ -108,7 +108,33 @@ Suppose you want to get rid of the gem-based theme and convert it to a regular t
 
 To do this, copy the files from the theme gem's directory into your Jekyll site directory. (For example, copy them to `/myblog` if you created your Jekyll site at `/myblog`. See the previous section for details.)
 
-Then remove references to the theme gem in `Gemfile` and configuration. For example, to remove `minima`:
+Then you must tell Jekyll about the plugins that were referenced by the theme. You can find these plugins in the theme's gemspec file as runtime dependencies. If you were converting the Minima theme, for example, you might see:
+
+```
+spec.add_runtime_dependency "jekyll-feed", "~> 0.9"
+spec.add_runtime_dependency "jekyll-seo-tag", "~> 2.1"
+```
+
+You should include this information in both your `_config.yml` file and `Gemfile`:
+
+```yaml
+# ./_config.yml
+
+plugins:
+  - jekyll-feed
+  - jekyll-seo-tag
+```
+
+```ruby
+# ./Gemfile
+
+group :jekyll_plugins do
```

Comment:
If using `group :jekyll_plugins`, the plugins do not need to be added to `_config.yml`

Reply:
*Note: GitHub Pages do not `require` plugins via Bundler. So adding to `_config.yml` is always better*

Reply:
Could be listed as an either/or.

Reply:
I think I understand. I've attempted to incorporate these notes. Please check for accuracy and style.